src/WinUI/ArcGIS.WinUI.Viewer/Samples/Data/ManageFeatures/ManageFeatures.xaml

```diff
@@ -0,0 +1,29 @@
+<UserControl 
+    x:Class="ArcGIS.WinUI.Samples.ManageFeatures.ManageFeatures"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:esri="using:Esri.ArcGISRuntime"
+    xmlns:esriUI="using:Esri.ArcGISRuntime.UI.Controls">
+    <Grid>
+        <esriUI:MapView x:Name="MyMapView" />
+        <Border Style="{StaticResource BorderStyle}">
+            <StackPanel>
+                <TextBlock Padding="5"
+                           Text ="Select an edit operation."
+                           FontWeight="SemiBold" />
+                <ComboBox x:Name="MethodPicker"
+                          HorizontalAlignment="Stretch"
+                          Padding="5"
+                          IsEditable="False"
+                          IsEnabled="True"
+                          SelectionChanged="OnComboBoxSelectionChanged" />
+                <TextBlock x:Name="InstructionLabel"
+                           Padding="5"
+                           Text ="Select an option to show instructions." />
+                <ComboBox x:Name="DamageTypeDropDown"
```

Comment:
We should include a label above this picker on `WinUI`, `WPF` and `MAUI` that says `Select damage type.` to indicate to the user what they are selecting.